test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/listModels.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var app = require('../../../../../');
+var async = require('async');
+var expect = require('chai').expect;
+var loopback = require('loopback');
+var path = require('path');
+var util = require('util');
+
+module.exports = function() {
+  var numberOfExpectedModels = 0;
+  var numberOfAvailableModels = 0;
+  var ModelConfig = app.models.ModelConfig;
+  var exampleWorkspace = path.resolve(__dirname, '../../../../../example');
+  var testsuite = this;
+
+  this.Given(/^I have a workspace containing (\d+) model\(s\)$/, function(numberOfExpectedModels, next) {
+    testsuite.numberOfExpectedModels = parseInt(numberOfExpectedModels);
+    next();
+  });
+
+  this.When(/^I list models for the workspace$/, function(next) {
+    ModelConfig.find(exampleWorkspace, function(err, list) {
```

Comment:
Move `var ModelConfig = app.models.ModelConfig;` here because you only use it here.